assets/multiqc_config.yaml

```diff
@@ -1,11 +1,40 @@
+custom_logo: "nf-core-rnafusion_logo_light.png"
+custom_logo_url: https://github.com/nf-core/rnafusion
+custom_logo_title: "nf-core/rnafusion"
+
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/rnafusion" target="_blank">nf-core/rnafusion</a>
+  This report has been generated by the <a href="https://github.com/nf-core/rnafusion/releases/tag/dev" target="_blank">nf-core/rnafusion</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/rnafusion" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/rnafusion/dev/docs/output" target="_blank">documentation</a>.
 report_section_order:
-  software_versions:
+  "nf-core-rnafusion-methods-description":
     order: -1000
-  nf-core-rnafusion-summary:
+  software_versions:
     order: -1001
+  "nf-core-rnafusion-summary":
+    order: -1002
 
 export_plots: true
+
+run_modules:
+  - fastqc
+  - qualimap
+  - picard
+  - mosdepth
+
+module_order:
+  - fastqc:
+      name: "FastQC"
+      path_filters:
+        - "*.zip"
+  - picard:
+      name: "Picard"
+  - qualimap:
```

Comment:
remove?

Reply:
Added to https://github.com/nf-core/rnafusion/pull/407